### ISSUE TITLE
Create a card to explain the changes with removal of the "good second issue" labels

### ISSUE THREAD

**Issue by jdingeman** [hackforla/website#4510]
### Dependency
- [x] #4423
- [x] #4432 is ready for implementation

### Overview
We need to add a card to the `Onboarding infos/links` column that explains the changes for removing the `Complexity: good second issue` label so that developers are aware of the change and don't try to look for good second issues.

### Action Items
- [x] Add a new card to the [Onboarding infos/links column](https://github.com/hackforla/website/projects/7#column-11381714)
  - [x] Populate the card with the following text:
  
````
#### [Card] Removal of "Complexity: good second issue" label
Dev leads are no longer creating good second issues. If you are a new dev and need to work on new issues, please pick up two good first issues.

Be sure to only work on one at a time, and wait for your first issue to be merged before picking up a second one. 
  ````
- [x] When you have created the new card, notify Merge Team to review and close this issue

#### For Merge Team/Tech Lead/PM
- [x] When this issue is closed, release the dependency for this issue on #4432

### Resources/Instructions
- This is part of epic #4432 
- HfLA site [Project Board](https://github.com/hackforla/website/projects/7)

**Comment by jdingeman** [comment]
@ExperimentsInHonesty - making this one a small issue since I think we'd want new devs to work on dev environment set up and git stuff before trying to make changes to the board.

**Comment by ExperimentsInHonesty** [comment]
@jdingeman is this a temporary card, until the roll out plan is complete and everything updated.  And if so, where is the issue to remove this card once its no longer needed.

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe I just want to confirm before I prioritize this, that there is an issue to remove it, once the roll out plan is done.

**Comment by roslynwythe** [comment]
- @ExperimentsInHonesty Here is the issue to remove the card: #5587

**Comment by segbuniwe** [comment]
Availability: I will be busy this weekend, but I will have about 6 hours during the week.
ETA: EOD Wednesday 10/25

**Comment by segbuniwe** [comment]
I will have this completed by EOD tomorrow 10/28 if no problems arise.

**Comment by segbuniwe** [comment]
@JessicaLucindaCheng 
Hi Jessica, for this issue a merge team member needs to review and close it. I have made the card needed for the onboarding section.

**Comment by JessicaLucindaCheng** [comment]
@segbuniwe Hi :smile:. I'm not a current member of the Merge Team. I suggest the following to reach out to the Merge Team: 
- @hackforla/website-merge in a new comment in this issue, and
- post a Slack message in the #hfla-site channel with a link to this issue.

In addition, move your issue into the _Questions /  In Review_ column and add a `ready for dev lead` label.

**Comment by segbuniwe** [comment]
@hackforla/website-merge team
Hi Merge Team, for this issue a merge team member needs to review and close it. I have made the card needed for the onboarding section.

**Comment by LRenDO** [comment]
Hi @segbuniwe! Great work! Thanks for working on this issue! I see that the card has been added in the appropriate column with the correct text.

For the future, when there is an issue like this one that doesn't result in a pull request, a review can be requested by moving it to the *Questions / In Review* column of the project board and add the `ready for dev lead` label. We regularly check that column and review those issues. 

Thanks again for contributing!
